lib/task/import/csvImportTask.class.php

```diff
@@ -1007,7 +1038,7 @@ public function execute($arguments = array(), $options = array())
             $this->log($e->getMessage(), sfLogger::ERR);
           }
         }
-        else if ($path = $self->rowStatusVars['digitalObjectPath'])
+        else if (($path = $self->rowStatusVars['digitalObjectPath']) && null === $self->object->getDigitalObject())
```

Comment:
I think the parentheses around ($uri = $self->rowStatusVars['digitalObjectPath']) are unnecessary.